www/gatsby-config.js

```diff
@@ -75,6 +77,64 @@ module.exports = {
         trackingId: 'UA-41298772-1',
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
+      options: {
+        query: `
+         {
+          site {
+            siteMetadata {
+              title
+              description
+              siteUrl
+              site_url: siteUrl
+            }
+          }
+        }`,
+        feeds: [
+          {
+            serialize: ({query: {site, allMarkdownRemark}}) => {
+              return allMarkdownRemark.edges.map(edge => {
+                return Object.assign(
+                  {},
+                  {
+                    title: edge.node.frontmatter.title,
+                    description: edge.node.html,
+                    date: edge.node.fields.date,
+                    url: site.siteMetadata.siteUrl + edge.node.fields.slug,
+                    guid: site.siteMetadata.siteUrl + edge.node.fields.slug,
+                  },
+                );
+              });
+            },
+            query: `
+              {
+                  allMarkdownRemark
+                  (limit: 10, 
+                  filter: {id: {regex: "/_posts/"}}, 
+                  sort: {fields: [fields___date], 
+                  order: DESC}) {
+                    edges {
+                      node {
+                        fields {
+                          date(formatString:"MMMM DD, YYYY, h:mm A")
+                          slug
+                        }
+                        frontmatter {
+                          title
+                        }
+                        excerpt
```

Comment:
What is the `excerpt` being used for? Looks like [the feed](https://deploy-preview-11003--reactjs.netlify.com/feed.xml) has descriptions that contain the entire article.

Reply:
Correct; I noticed that the previous RSS feed (http://web.archive.org/web/20170829050443/https://facebook.github.io/react/feed.xml) contained the whole article instead of just the excerpts, so that's why I kept it that way.

I'll remove the excerpt field in my next commit :)